src/SFA.DAS.AssessorService.Application.Api/Validators/ExternalApi/Certificates/GetBatchCertificateValidator.cs

```diff
@@ -26,48 +26,42 @@ public GetBatchCertificateRequestValidator(IStringLocalizer<GetBatchCertificateR
                 {
                     RuleFor(m => m).CustomAsync(async (m, context, cancellation) =>
                     {
-                        // NOTE: Currently we're making the Certificate & ILR record both mandatory
-                        var requestedIlr = await ilrRepository.Get(m.Uln, m.StandardCode);
-                        var sumbittingEpao = await organisationQueryRepository.GetByUkPrn(m.UkPrn);
-
-                        if (requestedIlr is null || !string.Equals(requestedIlr.FamilyName, m.FamilyName, StringComparison.InvariantCultureIgnoreCase))
-                        {
-                            context.AddFailure(new ValidationFailure("Uln", "Cannot find apprentice with the specified Uln, FamilyName & Standard"));
-                        }
-                        else if (sumbittingEpao is null)
+                        // NOTE: Currently we're making the Certificate & ILR record both mandatory - this is wrong fixing it!
+                        var submittingEpao = await organisationQueryRepository.GetByUkPrn(m.UkPrn);
+                       
+                        if (submittingEpao is null)
                         {
                             context.AddFailure(new ValidationFailure("UkPrn", "Specified UKPRN not found"));
                         }
                         else
                         {
-                            var providedStandards = await standardService.GetEpaoRegisteredStandards(sumbittingEpao.EndPointAssessorOrganisationId);
+                            var requestedIlr = await ilrRepository.Get(m.Uln, m.StandardCode);
+                            var existingCertificate = await certificateRepository.GetCertificate(m.Uln, m.StandardCode);
+                            var createdByCertificate = await certificateRepository.GetCertificateByOrgIdLastname(m.Uln, submittingEpao.EndPointAssessorOrganisationId, m.FamilyName);
```

Comment:
rename `createdByCertificate` to something like `existingCertificateCreatedByCallingEPAO` ? to make it obvious that this fact bypasses some validation

Reply:
yep. Makes sense. Will you be making the change @ben1stone ?